Source/WTF/wtf/Ref.h

```diff
@@ -336,8 +342,33 @@ inline RefPtr<match_constness_t<Source, Target>> dynamicDowncast(Ref<Source, Ptr
     return static_reference_cast<match_constness_t<Source, Target>>(WTFMove(source));
 }
 
+template<typename T> struct RefDerefTraitsWithoutDeletionCheck {
+    static ALWAYS_INLINE T* refIfNotNull(T* ptr)
+    {
+        if (LIKELY(ptr))
+            ptr->refWithoutDeletionCheck();
```

Comment:
"without deletion check" doesn't quite convey the semantics to me.
How about refAllowingDeletionToHaveBegun or refAllowingDeletionHasBegun?

Reply:
Right, naming is hard. `refAllowingDeletionToHaveBegun` sounds like to me, clearer for sure.

Reply:
Updated the name to `m_addedOrNewlyRenderedNodes`